code/modules/mob/living/carbon/human/human.dm

```diff
@@ -768,7 +768,7 @@
 
 	if(href_list["secrecordComment"])
 		if(hasHUD(usr,"security"))
-			if(usr.incapacitated())
+			if(usr.incapacitated() == !isobserver(usr)) //give the ghosts access to "View Comment Log" while they can't manipulate it
```

Comment:
Actually, hold on, I'm overlooking a more obvious solution here: `usr.incapacitated() && !isobserver(usr)`, since ghosts don't ever change their incapacitated state (and if they ever lost incapacitation somehow I don't see why they should stop being able to see this)

Reply:
True that, better so. Changed